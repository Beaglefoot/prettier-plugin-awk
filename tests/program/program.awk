@include "a"
@include "b"

function fn() {}
function fn2() {}
# comment
function fn2() {}
BEGIN{}
{}
END{}
