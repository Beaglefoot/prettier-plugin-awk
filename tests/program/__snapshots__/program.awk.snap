@include "a"
@include "b"

function fn() {}
function fn2() {}
# comment
function fn2() {}
BEGIN{}
{}
END{}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@include "a"
@include "b"

function fn() {}

function fn2() {}

# comment
function fn2() {}

BEGIN {}

{}

END {}
